source/AI.cpp

```diff
@@ -2760,6 +2769,41 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
+bool AI::DoSecretive(Ship &ship, Command &command)
+{
+	shared_ptr<Ship> scanningShip;
+	// Figure out if any ship is currently scanning us. If that is the case, move away from it.
+	for(auto otherShip : GetShipsList(ship, false))
```

Comment:
Other places in AI that iterate over the results of GetShipsList look to typically do `auto &other` or `auto &&other`, or in one case `auto *other`, but not just nothing.